packages/backend/src/studio.ts

```diff
@@ -276,7 +276,7 @@ export class Studio {
     this.#inferenceManager.init();
     this.#extensionContext.subscriptions.push(this.#inferenceManager);
 
-    /** The InstructLab tunning sessions manager */
+    /** The InstructLab running sessions manager */
```

Comment:
```suggestion
    /** The InstructLab tuning sessions manager */
```

Reply:
Thank you, should be resolved in the latest push